easybuild/easyblocks/t/tensorflow.py

```diff
@@ -342,6 +342,21 @@ def build_step(self):
         if self.toolchain.options.get('pic', None):
             cmd.append('--copt="-fPIC"')
 
+        # include install location of Python packages in $PYTHONPATH,
+        # and specify that value of $PYTHONPATH should be passed down into Bazel build environment;
+        # this is required to make sure that Python packages included as extensions are found at build time;
+        # see also https://github.com/tensorflow/tensorflow/issues/22395
+        pythonpath = os.getenv('PYTHONPATH', '')
+        env.setvar('PYTHONPATH', '%s:%s' % (os.path.join(self.installdir, self.pylibdir), pythonpath))
```

Comment:
Use os.pathjoin instead of %s:%s ?

Reply:
You probably mean `os.pathsep.join`? If so, done

Reply:
Yeah, can never remember the exact name of all these functions :-)